rviz_default_plugins/src/rviz_default_plugins/displays/effort/effort_display.cpp

```diff
@@ -196,8 +196,16 @@ void EffortDisplay::topic_callback(const std_msgs::msg::String & msg)
     if (joint->type == urdf::Joint::REVOLUTE || joint->type == 2) {
       std::string joint_name = it->first;
       urdf::JointLimitsSharedPtr limit = joint->limits;
-      joints_[joint_name] = std::make_shared<JointInfo>(joint_name, joints_category_);
-      joints_[joint_name]->setMaxEffort(limit->effort);
+      if (limit) {
+        joints_[joint_name] = std::make_shared<JointInfo>(joint_name, joints_category_);
+        joints_[joint_name]->setMaxEffort(limit->effort);
+      } else {
+        RCLCPP_WARN(
+          context_->getRosNodeAbstraction().lock()->get_raw_node()->get_logger(),
+          "Joint `%s` has no <limit> tag in URDF. Effort plugin needs to know the effort "
```

Comment:
```suggestion
          "Joint '%s' has no <limit> tag in URDF. Effort plugin needs to know the effort "
```

(it is just more similar to what we do elsewhere)